cdbot/cogs/fun.py

```diff
@@ -215,6 +215,12 @@ def check(reaction, user):
         if "Welcome to the Cyber Discovery" in message.content and message.author.id == WELCOME_BOT_ID:
             await message.add_reaction("\N{WAVING HAND SIGN}")
 
+    @Cog.listener()
+    async def on_reaction_add(self, reaction: Reaction, user: Member):
+        if str(reaction.message) == "\N{THUMBS DOWN SIGN}" and reaction.message.channel.id == QUOTES_CHANNEL_ID:
```

Comment:
`reaction.message` will give you the quote. `reaction.emoji` will give you the emoji.
```suggestion
        if str(reaction.emoji) == "\N{THUMBS DOWN SIGN}" and reaction.message.channel.id == QUOTES_CHANNEL_ID:
```